cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,49 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] PATH|RESOURCE_ALIAS(with resource_id)",
```

Comment:
I think we should be a bit more aligned with common documentation practices around command lines.

I propose two possible approaches:

One approach is listing the alternative uses:
```
delete [flags] PATH
delete [flags] RESOURCE_ALIAS RESOURCE_ID
```

The other is documenting it in a single line:
```
delete [flags] {PATH|RESOURCE_ALIAS RESOURCE_ID}
```

Reply:
I would suggest following the convention used by `oc` openshift CLI

Reply:
`oc` / `kubectl` seems to use the single line approach (at least quickly checking some examples in there like `oc label`), with a different character for grouping:

I think it should be in the following way then:
```
delete [flags] (PATH | RESOURCE_ALIAS RESOURCE_ID)
```